app/js/utils/DateTime.js

```diff
@@ -0,0 +1,75 @@
+/**
```

Comment:
Have you locked at using something like [moment.js](http://momentjs.com/) instead of rolling you own?